src/sc_data/constants.py

```
@@ -1,3 +1,17 @@
-DB_URL = "https://cdn.sparecores.net/sc-data/sc-data-all.sql.xz"
+CDN_BASE = "https://cdn.sparecores.net/sc-data"
+
+
+def _default_db_url():
+    """Use crawler major.minor for versioned CDN path when available; otherwise 'latest'."""
+    try:
+        import sc_crawler
+        v = sc_crawler.__version_info__
+        version = f"{v[0]}.{v[1]}"
+        return f"{CDN_BASE}/{version}/sc-data-all.sql.xz"
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)